docs/extending/extending_draftail.md

```diff
@@ -68,7 +80,7 @@ For detailed configuration options, head over to the [Draftail documentation](ht
 
 -   The `type` is the only mandatory piece of information.
 -   To display the control in the toolbar, combine `icon`, `label` and `description`.
--   The controls’ `icon` can be a string to use an icon font with CSS classes, say `'icon': 'fas fa-user',`. It can also be an array of strings, to use SVG paths, or SVG symbol references for example `'icon': ['M100 100 H 900 V 900 H 100 Z'],`. The paths need to be set for a 1024x1024 viewbox.
+-   The controls’ `icon` can be a string to use an icon font with CSS classes, say `'icon': 'fas fa-user',`. It can also be a string, to use SVG paths, or SVG symbol references for example `'icon': 'M100 100 H 900 V 900 H 100 Z',`. The paths need to be set for a 1024x1024 viewbox.
```

Comment:
I think we should try to keep the documentation as is here. If you re-read it says that we use the value as a path if it's an array.

Reply:
Oh, I see. Alright, I will revert the changes. Thanks